app/Phragile/Task.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Phragile;
+
+class Task {
+	private $title;
+	private $priority;
+	private $points;
+	private $status;
+	private $closed;
+	private $id;
+	private $assigneePHID;
+	private $assigneeName;
+	private $cssClass;
+
+	function __construct($attributes)
```

Comment:
If I didn't miss any place where the instance of this class is being created, these `$attributes` are always an array. We could have a type hint here then.